snippets/terraform-modification-disclaimer.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/terraform/rosa-sts-creating-a-cluster-quickly-terraform.adoc
+// * rosa_planning/rosa-understanding-terraform.adoc
+
+:_mod-docs-content-type: SNIPPET
+[IMPORTANT]
+====
+Because Terraform uses declarative language to change the current state of a resource, you should use caution when modifying any resources created with Terraform, such as performing upgrades. 
```

Comment:
Use caution when using resources outside of Terraform, such as the AWS console or Red Hat console, to modify resources created by Terraform. Using tools outside Terraform to manage resources that are already managed by Terraform introduces configuration drift from your declared Terraform configuration.